packages/yield-router/scripts/updateState.ts

```diff
@@ -1,57 +1,41 @@
-import { BuyBurnFixedClient } from "../client";
-import * as anchor from "@coral-xyz/anchor";
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { YieldRouterClient } from "../client";
 import { PublicKey } from "@solana/web3.js";
-import { getAssociatedTokenAddressSync } from "@solana/spl-token";
 
-/** Adjust these values to whatever you want them to be */
-const PRICE = 1;
-const PURCHASE_THRESHOLD = 100;
-const PURCHASE_PROPORTION = 0;
-
-const defaultTreasuryKey = "ALhQPLkXvbLKsH5Bm9TC3CTabKFSmnXFmzjqpTXYBPpu";
-const treasuryKey = new PublicKey(
-  process.env.TREASURY_KEY ?? defaultTreasuryKey
-);
-
-const defaultAuthority = "5HnwQGT79JypiAdjdjsXEn1EMD2AsRVVubqDyWfyWXRv";
-const authorityKey = new PublicKey(
-  process.env.AUTHORITY_KEY ?? defaultAuthority
+// mainnet Sunrise
+const defaultSunriseStateAddress =
+  "43m66crxGfXSJpmx5wXRoFuHubhHA1GCvtHgmHW6cM1P";
+const sunriseStateAddress = new PublicKey(
+  process.env.STATE_ADDRESS ?? defaultSunriseStateAddress
 );
 
-// used in devnet
-const defaultMint = "tnct1RC5jg94CJLpiTZc2A2d98MP1Civjh7o6ShmTP6";
-const mint = new PublicKey(process.env.MINT ?? defaultMint);
+// mainnet offset bridge wrapped SOL ATA
+const outputYieldAddresses = [
+  new PublicKey("4XTLzYF3kteTbb3a9NYYjeDAYwNoEGSkjoqJYkiLCnmm"),
+];
+const spendProportions = [100];
 
-const defaultHoldingAccount = "A4c5nctuNSN7jTsjDahv6bAWthmUzmXi3yBocvLYM4Bz";
-const holdingAccount = new PublicKey(
-  process.env.HOLDING_ACCOUNT ?? defaultHoldingAccount
-);
-
-// used for devnet testing
-const defaultStateAddress = "9QxfwoxkgxE94uoHd3ZPFLmfNhewoFe3Xg5gwgtShYnn";
-const stateAddress = new PublicKey(
-  process.env.STATE_ADDRESS ?? defaultStateAddress
-);
+// new update authority
+// very dangerous, isn't it?
+// we can set the authority to some address where we don't have the private key
+// so we can never change the authority ever again
+const newUpdateAuthority = new PublicKey("");
```

Comment:
I think it's maybe not necessary to include this update in this script here. We are not going to want to always update the authority with a new keypair.

Reply:
Or maybe we can allow it to be defined externally (e.g. a `process.env.NEW_UPDATE_AUTHORITY`) and skip this update when a new authority is not provided?